web/js/util/load.js

```diff
@@ -1,7 +1,9 @@
-var wv = wv || {};
-wv.util = wv.util || {};
+import $ from 'jquery';
+import { size } from 'lodash';
+import { default as uiUtils } from '../ui/ui.js';
+import { default as brand } from '../brand.js';
 
-wv.util.load = wv.util.load || (function () {
+export default (function () {
```

Comment:
Rename this function to `utilLoad`; then do imports as just `utilLoad` instead of `default as utilLoad`